/**:
  ros__parameters:
    clipping_distance: 1400